src/store/users/selectors.ts

```diff
@@ -1,5 +1,11 @@
-import { RootState } from '../reducer';
+import { call, select } from 'redux-saga/effects';
+import { getZEROUsers as getZEROUsersAPI } from '../channels-list/api';
 
-export function userByMatrixId(state: RootState, matrixId: string) {
-  return Object.values(state.normalized['users'] || {}).find((u: any) => u.matrixId === matrixId);
+export function* userByMatrixId(matrixId: string) {
```

Comment:
I don't think we should change this selector. It should remain strictly a selector. If we need something that reaches out to multiple places we can write a helper  function to use this and then fall back to an api request.

Reply:
will update